mysite/search/views.py

```diff
@@ -68,7 +67,15 @@ def search_index(request, invalid_subscribe_to_alert_form=None):
 
     format = request.GET.get('format', None)
     start = int(request.GET.get('start', 1))
-    end = int(request.GET.get('end', 10))
+
+    # If user selects number of bugs from drop-down list then 'limit' is the
+    # number of bugs to display obtained by POST otherwise display 10 bugs
+    if request.method == 'POST':
+        limit =  int(request.POST.get('items'))
```

Comment:
PEP8 complaint: `mysite/search/views.py:74:16: E222 multiple spaces after operator`

You have a stray space before `int`.